### PR TITLE
VIA Support: 1upkeyboards 1up60hte

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description

Adding the VIA keymap for the 1upkeyboards 1up60hte. I am using the same Vendor ID as what was used in the 1up60hse. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/1upkeyboards/1up60hte/config.h [hunk 1]
```diff
@@ -21,7 +21,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
+#define VENDOR_ID       0x6F75 // OU
 #define PRODUCT_ID      0x6060
```
Comment (noroadsleft) [inline keyboards/1upkeyboards/1up60hte/config.h]:
```suggestion
#define PRODUCT_ID      0x6874 // HT
```

?

Comment (mechmerlin) [reply keyboards/1upkeyboards/1up60hte/config.h]: why are we changing the product id?

Comment (noroadsleft) [reply keyboards/1upkeyboards/1up60hte/config.h]: I figured the HSE got a selected one, so why not do the same for this one?

Comment (mechmerlin) [reply keyboards/1upkeyboards/1up60hte/config.h]: HSE got one cuz its product id was like all 0000 if I remember correctly, but this one is directly from the 1upkeyboard kbfi****** json file.

Comment (noroadsleft) [reply keyboards/1upkeyboards/1up60hte/config.h]: KBF JSON files don't include preset USB Device ID data.

If the 1up60RGB gets VIA support added, its Product ID will have to be changed (it's currently identical to the HTE's).

Comment (mechmerlin) [reply keyboards/1upkeyboards/1up60hte/config.h]: alright good enough reason for me.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**